seq_strstr/host/dpu_host.c

```diff
@@ -0,0 +1,162 @@
+#include <dpu.h>
+#include <dpu_log.h>
+#include <dpu_memory.h>
+#include <dpu_program.h>
+#include <assert.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <fcntl.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <time.h>
+#include <sys/time.h>
+
+#include "dpu_host.h"
+#include "common.h"
+// dpu binary location TBD
+#define DPU_BINARY "build/strstr_dpu"
+#define DPU_LOG_ENABLE 1
+#define WRITE_OUT 1
+#define ALIGN(_p, _width) (((unsigned int)_p + (_width-1)) & (0-_width))
+
+
+
+dpu_error_t dpu_copy_to_dpu(struct dpu_set_t dpu, const char *symbol_name, uint32_t symbol_offset, const void *src, size_t length){
```

Comment:
This looks exactly the same as dpu_copy_to() from the SDK. Why do you need this function?

Reply:
When I first started multi-dpu approach, I didn't know that dpu_copy_to() can copy to a specific DPU, and now I just leave is as a reference.